contrib/endpoints/src/api_manager/config_manager.cc

```diff
@@ -164,15 +143,16 @@ void ConfigManager::fetch_configs(std::shared_ptr<ConfigsFetchInfo> fetchInfo) {
                          const std::string& config) {
```

Comment:
There was no problem with the code while I was compiling.
And I have checked addresses of c_str() of body in HTTPRequest callback,  config in the callback in GetConfig and string stored in fetchInfo->configs. All show the same memory address. That's why I thought it was working.

std::cout << __FILE__  << ":" << __LINE__ << " " << (long long)body.c_str() << std::endl;
std::cout << __FILE__  << ":" << __LINE__ << " " << (long long)config.c_str() << std::endl;
std::cout << __FILE__  << ":" << __LINE__ << " " << (long long)(fetchInfo->configs[fetchInfo->configs.size()-1].first.c_str()) << std::endl;

contrib/endpoints/src/api_manager/service_management_fetch.cc:90 32927672
contrib/endpoints/src/api_manager/config_manager.cc:147 32927672
contrib/endpoints/src/api_manager/config_manager.cc:153 32927672

But it seems that && is the right expression, I have updated it.